styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss

```diff
@@ -9,6 +9,15 @@
   position: relative;
   text-decoration: none;
   width: 100%;
+}
+
+.serviceLevelTeaser {
+  @include service-level-card;
+}
+
+// only add a hover class if a link
+.serviceLevelTeaser-link {
+  @include service-level-card;
```

Comment:
Sorry if I didn't clarify before.   I don't think we need the mixin here.  I'd say leave `.serviceLevelTeaser` as-is.  
Then that additional class `.serviceLevelTeaser-link` only has the link-specific behavior, and then in the HTML we would have something like this:

```
<a class="serviceLevelTeaser serviceLevelTeaser-link" href="#"></a>
```